apiserver/common/modelmanagerinterface.go

```diff
@@ -116,7 +117,13 @@ func (st modelManagerStateShim) NewModel(args state.ModelArgs) (Model, ModelMana
 	if err != nil {
 		return nil, nil, err
 	}
-	return modelShim{m}, modelManagerStateShim{otherState, st.pool}, nil
+	otherModel, err := otherState.Model()
```

Comment:
The model result from NewModel() above is the otherModel - why make the same API call again here?